hack/testing/test-0530-bz1667801-fix-kibana-replica-shards

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../lib/init.sh"
```

Comment:
you don't need the `source` here since `0530-bz1667801-fix-kibana-replica-shards` does it